### DISCUSSION TITLE
Is there a way to temporarily disable explorerpatcher?

### DISCUSSION THREAD

**Discussion by wikibamxd** [valinet/ExplorerPatcher#3604] [Q&A]
Im trying to return to a game that i didnt play in a long time and everytime i tried starting it i would receive a "suspicious program" warning, and i tried with everything that i had and the one causing it was explorer patcher.. ( the game is digimon masters online)
![2024-08-29 09_45_37-Digimon Masters - Returning Player Guide - YouTube — Mozilla Firefox](https://github.com/user-attachments/assets/f4b3bbf2-ce92-4d96-9e10-b0e9cfac5bd7)

**Comment by pyrates999** [comment]
try this:

1. rename C:\Windows\dxgi.dll to C:\Windows\dxgi.dll.old
2. open up task manager
3. kill explorer.exe
4. run explorer.exe

when you want to re-enable:

1. rename C:\Windows\dxgi.dll.old to C:\Windows\dxgi.dll
2. open up task manager
3. kill explorer.exe
4. run explorer.exe

**Reply by wikibamxd** [reply]
did exactly how you said. and the game still said the same thing

**Reply by wikibamxd** [reply]
and i know its explorer patcher bcs i unninstalled it and tested to see, and the game worked fine without it

**Reply by pyrates999** [reply]
There's not much else you can do then as the game is checking for this, sorry.  It's an anti-cheat measure.

**Reply by regs01** [reply]
> There's not much else you can do then as the game is checking for this, sorry. It's an anti-cheat measure.

Well, a Disable/Enable button in ExplorerPatcher settings would be useful sometimes.

**Reply by pyrates999** [reply]
I'll let the developers decide on this.  But with how it is integrated into windows, I don't think a simple enable/disable option would work.

**Reply by wikibamxd** [reply]
thank you though

**Reply by pyrates999** [reply]
you're welcome

**Reply by ahmadalam** [reply]
Windows 11, version 24H2 blocks installation until it's disabled, which is why I came here to find out how. However renaming the C:\Windows\dxgi.dll file did not work; needed an admin account to do this. Restarted Explorer a good few times, but it wouldn't budge. In the end, I had to uninstall it.

**Reply by pyrates999** [reply]
Yes, you have to uninstall EP before you upgrade to 24h2.  Which this discussion was not about.  This was about temporarily disabling EP for other reasons.

**Reply by ahmadalam** [reply]
Not needed: "Which this discussion was not about.", title states otherwise "Is there a way to temporarily disable explorerpatcher?". I wanted to disable it temporarily so I could get past the 24H2 update dialogue box wanting it to be shut down. Anyway, may not reinstall it, I work across 4 PC's and keeping it updated is now simply too much hassle, turn on one PC after a few days and you're prompted that it's been disabled because it keeps on crashing etc.

**Reply by Amrsatrio** [reply]
@ahmadalam That's the way it is today. Microsoft keeps removing features that EP resurrects. We fight back my adapting our patches to their updates. The problem is, Windows updates automatically monthly while EP doesn't. If I add auto updates to EP, I don't trust myself like if someone happens to hijack my GitHub account or other maintainers', it could be a large scale IT meltdown.

So sorry that it happened to be the situation today. It is kind of expected considering those are Windows 10 remnants that should not be there in 11 in the first place.

**Reply by ahmadalam** [reply]
Sad but true. Microsoft shouldn't have such a dictatorial attitude, you have my deepest sympathies, good luck with your endeavour.

**Comment by btmacdou** [comment]
**1/29/25**
I also wanted to be able to enable and disable Explorer Patcher.  As of this post I am still running version 23H2.  Not sure when I will be forced to upgrade.  In the meantime, I took the advice of @pyrates999 (top of the reply list) and wrote a batch file to rename the dxgi.dll file to  dxgi.dll.tmp (and back).  The batch file allows you to enable and disable Explorer Patcher by alternating the name of the file from the correct name to a TMP name.  

Unfortunately, I could not find a way to automate the entire process because using the "Taskkill" command to terminate Exlorer.exe in a batch file results in a blank screen and terminates the batch file.  I tried multiple ways to get around this but am not smart enough with windows programming to figure it out. 

**However**, it's easy to get out of the blank screen using CTRL-ALT-DEL which brings up a menu that allows you to reboot, log off, OR RUN TASK MANAGER.  And this is the trick.  _From Task Manager, it's simple to restart explorer using the "**Run New Task**" button_

And it's actually very quick and easy to run Explorer from Task Manager and switch EP in and out WITHOUT REBOOTING OR LOGGING OFF!

**The only caveat is that the batch file must be run in administrator mode**.  There are two ways to do this:

1. When you launch it, right click the file and choose "Run as Administrator"
2. Create a shortcut on your desktop and set it's properties to Run as Administrator.  (This way there is no right click required).

These options assume that your login account has administrator privileges.  If you are using a work computer, there is not much chance of that.  But if it's your own personal computer then your login account can be configured to belong to the administrator group.

Here is my batch file saved as a TEXT file.  You can rename it back to a .bat file and run it or edit it with Notepad as you see fit:
[Explorer Patcher Enable-Disable.bat.txt](https://github.com/user-attachments/files/18595900/Explorer.Patcher.Enable-Disable.bat.txt)

From the blank screen that you get after killing EP, hit CTRL-ALT-DEL and select Task Manager, then restart Explorer:

![image](https://github.com/user-attachments/assets/2d8e2204-7938-4998-845b-4497e2c94bae)

**Reply by z-impi** [reply]
for QOL i changed:
"echo` 4 Kill Explorer Task (press CTRL-ALT-DEL to access Task Manager)"
to
"echo 4 Kill Explorer Task (press CTRL-SHIFT-ESC to access Task Manager)"

This opens the task manager directly instead of the task menu where you have to select the task manager.
Thank you for your time and effort to put this together. 

* * * * *

Also, Microsoft created this registry entry on my system while the setup scanned for incompatible software. 
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\WindowsUpdate\UX\Settings
DWORD = "IsBlockerActive"

You have to delete it or set it to 0. Removing explorer patcher after the update attempt results in this entry (sometimes).
Hope that helps someone in need.
